app/src/main/java/org/simple/clinic/summary/medicalhistory/MedicalHistorySummaryEffectHandler.kt

```diff
@@ -38,7 +40,14 @@ class MedicalHistorySummaryEffectHandler @AssistedInject constructor(
   private fun loadMedicalHistory(): ObservableTransformer<LoadMedicalHistory, MedicalHistorySummaryEvent> {
     return ObservableTransformer { effects ->
       effects
-          .switchMap { medicalHistoryRepository.historyForPatientOrDefault(it.patientUUID).subscribeOn(schedulers.io()) }
+          .switchMap {
+            medicalHistoryRepository
```

Comment:
Should we change this to `medicalHistoryRepository.historyForPatientOrDefaultImmediate`, WDYT?

Reply:
We could, I decided not to do it since it's not related to this PR. This PR was about being able to fix the `UUID` instances being generated. We can make these changes when we have to add new functionality to this class in the future.